blueprints/addon/files/.travis.yml

```diff
@@ -13,8 +13,8 @@ sudo: false
 <% } %>
 env:
   # we recommend testing LTS's and latest stable release (bonus points to beta/canary)
-  - EMBER_TRY_SCENARIO=ember-lts-2.4
```

Comment:
I don't know. I think @kategengler wanted to change this anyways to not actually use different Travis envs, so maybe we should wait for that to reduce blueprint churn?

/cc @rwjblue